packages/fx-core/resource/package.nls.json

```diff
@@ -843,8 +843,8 @@
   "error.BadComponent": "Component '%s' lacks of property: %s.",
   "error.InvalidFeature": "This feature is not applicable for your project.",
   "error.UpdateAadManifest.MissingEnvHint": "If you are developing with a new project created with Teams Toolkit, running provision or debug will register correct values for these environment variables.",
-  "driver.aadApp.description.create": "Create an Azure Active Directory app to authenticate users",
-  "driver.aadApp.description.update": "Apply Azure Active Directory app manifest to an existing app",
+  "driver.aadApp.description.create": "Create an Azure Active Directory app to authenticate users.",
+  "driver.aadApp.description.update": "Apply Azure Active Directory app manifest to an existing app.",
```

Comment:
If this is really about appending an AAD app manifest to existing app, can replace "Apply" with "Add"

Reply:
In this case, we will use the aad manifest to update existing app. When creating aad app, we have create the aad manifest  for it. So it is not a new aad manifest.